scripts/buildorder.py

```diff
@@ -73,27 +76,31 @@ def parse_build_file_excluded_arches(path):
     with open(path, encoding="utf-8") as build_script:
         for line in build_script:
             if line.startswith(('TERMUX_PKG_BLACKLISTED_ARCHES', 'TERMUX_SUBPKG_EXCLUDED_ARCHES')):
-                arches_string = line.split('ARCHES=')[1]
-                for char in "\"'\n":
-                    arches_string = arches_string.replace(char, '')
+                arches_string = remove_char(line.split('ARCHES=')[1])
                 for arches_value in re.split(',', arches_string):
                     arches.append(arches_value.strip())
 
     return set(arches)
 
-def parse_build_file_variable_bool(path, var):
-    value = 'false'
+def parse_build_file_variable(path, var):
+    value = None
 
     with open(path, encoding="utf-8") as build_script:
         for line in build_script:
             if line.startswith(var):
-                value = line.split('=')[-1].replace('\n', '')
+                value = remove_char(line.split('=')[-1])
                 break
 
-    return value == 'true'
+    return value
+
+def parse_build_file_variable_bool(path, var):
+    return parse_build_file_variable(path, var) == 'true'
 
 def add_prefix_glibc_to_pkgname(name):
-	return name.replace("-static", "-glibc-static") if "static" == name.split("-")[-1] else name+"-glibc"
+    return name.replace("-static", "-glibc-static") if "static" == name.split("-")[-1] else name+"-glibc"
```

Comment:
I know this is a string concatenation, but please use whitespace around operators.
```suggestion
    return name.replace("-static", "-glibc-static") if "static" == name.split("-")[-1] else name + "-glibc"
```